### ISSUE TITLE
Test for Web Accessibility

### ISSUE THREAD

**Issue by loket** [cruftlab/tilfeldigeord.no#69]
It should be relatively simple to make the site compliant in terms of accessibility.

Here be tools: https://a11y.coffee/start-testing/

**Comment by loket** [comment]
Here's a GitHub Action that runs Lighthouse and posts the results as a badge in the readme: https://github.com/marketplace/actions/lighthouse-badger
